Lib/sysconfig.py

```diff
@@ -665,7 +665,15 @@ def get_platform():
             machine += ".%s" % bitness[sys.maxsize]
         # fall through to standard osname-release-machine representation
     elif osname[:3] == "aix":
-        return "%s-%s.%s" % (osname, version, release)
+        # bootstrap: during build _posixsubprocess may be unavailable
```

Comment:
Side note: this is duplicated in two modules. It might make sense to apply DRY in the future...

Reply:
apply DRY ? not clear what that means.

Reply:
DRY == Don't Repeat Yourself == put repeated code into shared functions. But this probably shouldn't be a part of this PR.

Reply:
Perhaps not as elegant, I stole the idea from the `macos` comments. Maybe, now that I using yet another piece of `subprocess` - the comment is unneeded. I'll test and comment.